storage/tests/unit/test_notification.py

```diff
@@ -16,6 +16,72 @@
 
 import mock
 
+class TestProjectID(unittest.TestCase):
+
+    from google.cloud.storage import notification
+    PROJECT = notification._PROJECT_PATTERN
+
+    def test_min_length_fail(self):
+        import re
```

Comment:
Imports from the standard library are OK to do at module-level (i.e. they don't conflict with our testing style, though you'll find cases where standard library modules **are** imported at runtime and you'll find cases where they aren't)